From 3cd092ac5508993dc98bcf7ae32aea578cae9640 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Uwe=20Kleine-K=C3=B6nig=20=28The=20Capable=20Hub=29?=
 <u.kleine-koenig@baylibre.com>
Date: Fri, 17 Jul 2026 16:50:53 +0200
Subject: [PATCH 09288/15333] misc: mei: Use named initializers for
 acpi_device_id array
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

While being less compact, using named initializers allows to more easily
see which members of the structs are assigned which value without having
to lookup the declaration of the struct. And it's also more robust
against changes to the struct definition.

While touching this array, add a space to the list terminator to make it
match the most common usage.

Signed-off-by: Uwe Kleine-König (The Capable Hub) <u.kleine-koenig@baylibre.com>
Link: https://patch.msgid.link/c189ce8aeae288e9d8bcdfff150957c6f3e6e37d.1784299069.git.u.kleine-koenig@baylibre.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/misc/mei/vsc-tp.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/misc/mei/vsc-tp.c b/drivers/misc/mei/vsc-tp.c
index 5ecf99883996..432bc59f11f9 100644
--- a/drivers/misc/mei/vsc-tp.c
+++ b/drivers/misc/mei/vsc-tp.c
@@ -552,11 +552,11 @@ static void vsc_tp_remove(struct spi_device *spi)
 }
 
 static const struct acpi_device_id vsc_tp_acpi_ids[] = {
-	{ "INTC1009" }, /* Raptor Lake */
-	{ "INTC1058" }, /* Tiger Lake */
-	{ "INTC1094" }, /* Alder Lake */
-	{ "INTC10D0" }, /* Meteor Lake */
-	{}
+	{ .id = "INTC1009" }, /* Raptor Lake */
+	{ .id = "INTC1058" }, /* Tiger Lake */
+	{ .id = "INTC1094" }, /* Alder Lake */
+	{ .id = "INTC10D0" }, /* Meteor Lake */
+	{ }
 };
 MODULE_DEVICE_TABLE(acpi, vsc_tp_acpi_ids);
 
-- 
2.55.0

